03_ScienceExamples/MagellanicClouds/ExtremelyMetalPoorStarsInTheLMC.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<p>I would be great to explicitly reference the appropriate sections and/or papers for these selection criteria. </p><p>A link for SExtractor is broken. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>